src/object/trigger_manager.c

```diff
@@ -4225,6 +4232,24 @@ tr_find_trigger (const char *name)
 
   if (trigger_table_find (name, &object) == NO_ERROR)
     {
+      /* This is the case when the loaddb utility is executed with the --no-user-specified-name option as the dba user. */
+      if (db_get_client_type () == DB_CLIENT_TYPE_ADMIN_UTILITY && prm_get_bool_value (PRM_ID_NO_USER_SPECIFIED_NAME))
+	{
+	  if (object == NULL)
+	    {
+	      char other_trigger_name[DB_MAX_IDENTIFIER_LENGTH_287] = { '\0' };
+
+	      do_find_trigger_by_query (name, other_trigger_name, DB_MAX_IDENTIFIER_LENGTH_287);
+	      if (other_trigger_name)
```

Comment:
Do you intend to the following?
```suggestion
	      if (other_trigger_name[0] != '\0')
```

Reply:
I will change it.